csvcut.py

```diff
@@ -0,0 +1,103 @@
+from argparse import ArgumentParser
+import sys
+
+
+DESCRIPTION = 'csvpp - prints cvs file in human-readable format'
+EXAMPLES = 'example: cat file.txt | csvpp -f | less -SR'
+
+
+def write_row(index ,row, output_stream):
+    """
+    Write_row modifeis the rows, leaving only the column desired
+    :param index: the indeces of columns that need to remain in the file
+    :param row: row represented as a list of columns
+    :param output_stream: a stream of modified row
+
+    """
+    output_line = ''
+    for i, column in enumerate(row):
+        if(i in index):
+            if(i == max(index)):
+                output_line += column + ""
+            elif(i < max(index)):
+                output_line += column  + ','
+    
+    output_line += '\n'
+    output_stream.write(output_line)
+
+
+def main():
+    args = parse_args()
+    input_stream = open(args.file, 'r') if args.file else sys.stdin
+    output_stream = open(args.output_file, 'r') if args.output_file else sys.stdout
+
+    columns = input_stream.readline().strip().split(args.separator)
+    first_rows = [columns]
+    
+    index = []
+    fields_int = []
+
+    if args.fields == 'None':
```

Comment:
Please don't introduce 'None' string. What if there is a column named None? In python You have a special value called None. Use it.